official/resnet/resnet_run_loop.py

```diff
@@ -355,21 +370,35 @@ def resnet_main(flags, model_function, input_function, shape=None):
       allow_soft_placement=True)
 
   # Set up a RunConfig to save checkpoint and set session config.
-  run_config = tf.estimator.RunConfig().replace(save_checkpoints_secs=1e9,
-                                                session_config=session_config)
+  if not flags.use_distribution_strategy:
+    run_config = tf.estimator.RunConfig().replace(
+        save_checkpoints_secs=1e9, session_config=session_config)
+  else:
+    if flags.gpus_for_distribution_strategy == 0:
+      distribution = one_device_strategy.OneDeviceStrategy(
+          'device:CPU:0')
+    elif flags.gpus_for_distribution_strategy == 1:
+      distribution = one_device_strategy.OneDeviceStrategy(
+          'device:GPU:0')
+    else:
+      distribution = mirrored_strategy.MirroredStrategy(
+          num_gpus=flags.gpus_for_distribution_strategy
+      )
+    run_config = tf.estimator.RunConfig(distribute=distribution).replace(
```

Comment:
Current TF master implies the kwarg is now train_distribute; @guptapriya , what is the current format?

Reply:
ah yes, sorry we just changed it after a discussion. you're right, it's called `train_distribute` now. thanks for catching!

Reply:
Perils of working from contrib. ^^